commcare_connect/opportunity/views.py

```diff
@@ -1119,7 +1120,7 @@ def user_visit_review(request, org_slug, opp_id):
         review_status = request.POST.get("review_status").lower()
         updated_reviews = request.POST.getlist("pk")
         user_visits = UserVisit.objects.filter(pk__in=updated_reviews)
-        if review_status in ["agree", "disagree"]:
+        if review_status in [VisitReviewStatus.agree.value, VisitReviewStatus.disagree.value]:
```

Comment:
This might be out of scope for this PR, but why are we using `Agree/Disagree` terms as opposed to existing `Approved/Rejected` from VisiValidationStatus?

Reply:
That was done to reduce the confusion that came due to those choices. These `Agree/Disagree` choices were distinct enough from the `Approved/Rejected` choices.